ent/schema/token.go

```diff
@@ -0,0 +1,37 @@
+package schema
+
+import (
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent"
+	"entgo.io/ent/schema"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+)
+
+type Token struct {
+	ent.Schema
+}
+
+// Fields returns Balance fields.
+func (Token) Fields() []ent.Field {
+	return []ent.Field{
+		// contract:tokenId
+		field.String("id").Unique().Immutable(),
+		field.Uint64("tokenId"),
```

Comment:
Is this for ERC721? It would be useful to have a `type` enum here to disambiguate. In the case of ERC721, we should store the token uri as well